tests/test_parquet.py

```diff
@@ -2212,5 +2162,81 @@ def testRowGroupSizeDataFrameWithLists(self):
         self.assertGreater(row_group_size, 1_000_000)
 
 
+def _checkAstropyTableEquality(table1, table2, skip_units=False, has_bigendian=False):
```

Comment:
Now I wonder if this should replace `assertAstropyTablesEqual` in lsst.daf.butler.tests.utils.

Reply:
I hadn't noticed this!  Is it worth looking into merging these now?

Reply:
I'm sure your checks are more complete and so we may as well sync up the utility code. Not required now though.